Assets/Scripts/GameManager.cs

```diff
@@ -68,15 +85,47 @@ public class GameManager : MonoBehaviour {
 
     private void FillUITexts() {
         if (!currentTextAlreadyFilled) {
+            Statement CurrQuestion = CurrentQuestion;
+            Answer CurrAnswer;
```

Comment:
Please write "CurrentAnswer" instead of "CurrAnswer". Names should be as explicit as possible